.github/workflows/whitespace_checks.yml

```diff
@@ -0,0 +1,45 @@
+name: whitespace_checks
+
+# Check whitespaces of the following file types
+# Not checked: .lua, .yml, .properties, .conf, .java, .py, .svg, .gradle, .xml, ...
+# (luacheck already checks .lua files)
```

Comment:
Note that Luachecks only detects inconsistent indentation (space followed by tab). So luacheck won't warn about e.g. one line using tabs, another one using spaces.

Reply:
This is meant for trailing whitespaces, currently indentation aren't checked by the script anyway.